modules/rosa-accessing-your-cluster.adoc

```diff
@@ -3,13 +3,20 @@
 // getting_started_rosa/creating-first-rosa-cluster.adoc
```

Comment:
I believe this should be `getting_started_rosa/accessing-rosa-cluster.adoc`

Reply:
Done.